e2e/test/scripts/create_cluster.sh

```diff
@@ -5,54 +5,51 @@ set -o pipefail
 set -o nounset
 set -x
 
-export LINODE_API_TOKEN="$1"
+export LINODE_TOKEN="$1"
 export CLUSTER_NAME="$2"
-export K8S_VERSION="$3"
+export KUBERNETES_VERSION="$3"
+export CAPLI_VERSION="0.1.0"
+export WORKER_MACHINE_COUNT=1
+export LINODE_CONTROL_PLANE_MACHINE_TYPE=g6-standard-2
+export LINODE_MACHINE_TYPE=g6-standard-2
+export KUBECONFIG="$(realpath "$(dirname "$0")/../kind-management.conf")"
 
 if [[ -z "$4" ]]
 then
-  export REGION="eu-west"
+  export LINODE_REGION="us-sea"
 else
-  export REGION="$4"
+  export LINODE_REGION="$4"
 fi
 
-TEST_MANIFEST=$(realpath "$(dirname "$0")/../manifest/linode-blockstorage-csi-driver.yaml")
-
-cat > cluster.tf <<EOF
-variable "server_type_node" {
-  default = "g6-standard-2"
-}
-variable "nodes" {
-  default = 2
-}
-variable "server_type_master" {
-  default = "g6-standard-2"
-}
-variable "region" {
-  default = "$REGION"
-}
-variable "ssh_public_key" {
-  default = "${HOME}/.ssh/id_rsa.pub"
-}
-module "k8s" {
-  source  = "git::https://github.com/linode/terraform-linode-k8s.git?ref=master"
-  linode_token = "$LINODE_API_TOKEN"
-  cluster_name = "$CLUSTER_NAME"
-  csi_manifest = "file://${TEST_MANIFEST}"
-  server_type_node = "\${var.server_type_node}"
-  nodes = "\${var.nodes}"
-  server_type_master = "\${var.server_type_master}"
-  region = "\${var.region}"
-  ssh_public_key = "\${var.ssh_public_key}"
-  k8s_version = "$K8S_VERSION"
-}
-EOF
-
-terraform workspace new ${CLUSTER_NAME}
-
-terraform init
-
-terraform apply \
- -auto-approve
+METADATA_REGIONS="nl-ams in-maa us-ord id-cgk us-lax es-mad us-mia it-mil jp-osa fr-par br-gru us-sea se-sto us-iad"
+[[ "$METADATA_REGIONS" =~ .*"${LINODE_REGION}".* ]] || (echo "Given region doesn't support Metadata service" ; exit 1)
```

Comment:
I believe CAPL supports regions without metadata now. @eljohnson92 can you confirm?

Reply:
it does now with a stackscript shim

Reply:
@luthermonson do we want to support it?

Reply:
The way I understand it we shouldn't need the region check and CAPL will just work in any region when we pin and use the official release. Probably safe to remove this check and rely on CAPL completely but I'll let @eljohnson92 make the call since he wrote the feature.

Reply:
that is correct, CAPL is able to detect if it needs to use the stackscript shim or not automatically and should just work transparently in all regions